### PR TITLE
Update sidemenu.blade.php

### PR DESCRIPTION
**Pull request by AbbyJanke**
Add user's name and avatar preview in the account setting sidebar similar to how it is shown within AdminLTE Demo:

https://adminlte.io/themes/AdminLTE/pages/examples/profile.html

### CODE REVIEW

File: src/resources/views/auth/account/sidemenu.blade.php [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <div class="box">
+    <div class="box-body box-profile">
+
```
Comment (lloy0076) [inline src/resources/views/auth/account/sidemenu.blade.php]: What's that extra line for?

File: src/resources/views/auth/account/sidemenu.blade.php [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+	    <img class="profile-user-img img-responsive img-circle" src="{{ backpack_avatar_url(Auth::user()) }}">
```
Comment (lloy0076) [inline src/resources/views/auth/account/sidemenu.blade.php]: And if that `Auth::user()` returns `null`?

File: src/resources/views/auth/account/sidemenu.blade.php (hunk 1 above)
Comment (AbbyJanke) [reply src/resources/views/auth/account/sidemenu.blade.php]: Following the overall style of the rest of the file.

File: src/resources/views/auth/account/sidemenu.blade.php (hunk 2 above)
Comment (AbbyJanke) [reply src/resources/views/auth/account/sidemenu.blade.php]: If you look at sidebar.blade.php (which is where i copied the line from) it is exactly the same so look at @tabacitu for that answer.

Comment (lloy0076) [reply src/resources/views/auth/account/sidemenu.blade.php]: If it's `null` we'll get a PHP error; I guess seeing we're meant to be logged in here it won't ever be `null`.

**Comment by tabacitu** [comment]
Yup, totally agree with this. Just added another <hr> for style. Merged, thank you guys, cheers!

**Pull request merged**